doc/catalog-format-spec.md

```diff
@@ -267,7 +267,7 @@ static, i.e. they cannot depend on the context.
 
    ```toml
    project-files = ["my_project.gpr", "utils/utils_for_my_project.gpr"]
-
+   
```

Comment:
Looks like trailing white-spaces are introduced.

Reply:
Ugh. I used a new .md editor, that will be it.